src/components/Popup/styles.ts

```diff
@@ -0,0 +1,40 @@
+export const popupStyles = `
```

Comment:
This type of css stying is hard to maintain. Should we instead use a regular css and import it in the `popup.ts`